src/Umbraco.Headless.Client.Net/Management/ContentManagementService.cs

```diff
@@ -80,6 +82,7 @@ public ContentManagementService(IApiKeyBasedConfiguration configuration)
 
         public IContentService Content { get; }
         public IDocumentTypeService DocumentType { get; }
+        public IFormService FormService { get; }
```

Comment:
Wouldn't it be more consistent with the rest of the "services" if this was simply called Forms ?

Reply:
Must've been automplete 😅 I've changed it to `Forms`